lib/ability.rb

```diff
@@ -20,11 +20,13 @@ def initialize(user, resource, controller = nil)
         can :manage, AccountPriceGroupMember
       else
         can :manage, :all
+        cannot([:edit, :update], User) { |target_user| !target_user.external? }
```

Comment:
Would it make sense to make an `internal?` method that's just `!external?`? 

Also, part of me wonders if it wouldn't be better to make a new message like `User#admin_updatable?` or `details_updatable?` wrapping `external?` (similar to the `User#password_updatable?` method). We're likely going to be changing the definition of internal vs external users in regards to pricing for NU soon, so it would be nice to not have these concepts be quite so conflated (they already are a little bit, but we should tease them apart where possible).